app/models/concerns/thredded/post_common.rb

```diff
@@ -13,14 +13,18 @@ module PostCommon
 
       validates :content, presence: true
 
-      scope :order_oldest_first, -> { order(id: :asc) }
-      scope :order_newest_first, -> { order(id: :desc) }
+      scope :order_oldest_first, -> { order(created_at: :asc) }
```

Comment:
This should also sort by ID in case of conflicts to establish a stable ordering. `order(created_at: :asc, id: :asc)`